CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this
+repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your
+interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the
+   layer when doing a build.
```

Comment:
What does this mean?

Reply:
I think this refers to removing any files that are not part of the repo that may have been generated while you were editing or building the code. For example those swap files from vim or if for instance they did `bundle install --path=new_vendor/bundle`.